### PR TITLE
Create 0.17 dockerfile

### PR DESCRIPTION
**Pull request by nexusmaniac**
Updated to Alpine 3.9 & added preliminary support for the 0.17 headless binary (once released) 💥

### CODE REVIEW

File: 0.17/dockerfile [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+MAINTAINER https://github.com/dtandersen/docker_factorio_server
+
+ARG USER=factorio
+ARG GROUP=factorio
+ARG PUID=845
+ARG PGID=845
+
+ENV PORT=34197 \
+    RCON_PORT=27015 \
+    VERSION=0.17 \
```
Comment (jasonwc) [inline 0.17/dockerfile]: Do we need to specify the patch version here?

**Comment by jasonwc** [comment]
Thanks for jumping on this! Was about to come do the same. I'm excited to get 0.17 up on a cluster somewhere and start playing.

**Comment by nexusmaniac** [comment]
Same here! 😄 Can't wait for them to push the binary.

Gonna play some solo factorio in the mean time 😎 

Nice to see that this is a popular container haha!

**Comment by Jawastew** [comment]
headless server seems to be downloadable (under version 0.17.0 https://www.factorio.com/get-download/0.17.0/headless/linux64 )

**Comment by jasonwc** [comment]
Yep! Looks like @dtandersen is knocking out the rest of the changes on `master`

**Comment by Jawastew** [comment]
Yay :dancer:

**Comment by dtandersen** [comment]
Got stuck:

```
factorio_1  |    0.691 Goodbye
factorio_1  |    0.000  Error Util.cpp:75: "grass" is not a valid autoplace control name.
```

**Pull request merged**